test/integration/crud/misc_cursors.test.js

```diff
@@ -3589,7 +3584,7 @@ describe('Cursor', function () {
     await client.close();
   });
 
-  it('should return implicit session to pool when client-side cursor exhausts results after a getMore', function (done) {
+  it('should return implicit session to pool when client-side cursor exhausts results after a getMore', async function () {
     const configuration = this.configuration;
     const client = configuration.newClient({ w: 1 }, { maxPoolSize: 1 });
```

Comment:
This file has a shared client the test could be updated to use

Reply:
Done.